sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -21,8 +21,9 @@
 
 class TableServiceStats(GenTableServiceStats):
     """Stats for the service
+
     :param geo_replication: Geo-Replication information for the Secondary Storage Service.
-    :type geo_replication: ~azure_table.models.GeoReplication
+    :type geo_replication: ~azure.data.tables._generated.models.GeoReplication
```

Comment:
We shouldn't link the documentation to the `_generated` module.
We should just be able to use `~azure-data.tables.GeoReplication`